cmd/collect_signals/main.go

```diff
@@ -39,6 +41,58 @@ const (
 	githubAuthServerMaxAttemps = 10
 )
 
+const (
+	configDataset           = "dataset"
+	configDatasetTTLHours   = "dataset-ttl-hours"
+	configLocal             = "local"
+	configScoring           = "scoring"
+	configScoringConfigFile = "scoring-config"
+	configScoringColumnName = "scoring-column-name"
+)
+
+type runner interface {
+	Run() error
+}
+
+// getWorkLoop chooses between either scorecard's worker.WorkLoop implementation
+// or the internal localworker.WorkLoop implementation. The localworker.WorkLoop
+// is chosen if the "local" criticality config variable is enabled.
+func getWorkLoop(logger *zap.Logger, criticalityConfig map[string]string, w worker.Worker) (runner, error) {
+	local, err := parseBool(criticalityConfig[configLocal], false)
+	if err != nil {
+		return nil, fmt.Errorf("parsing %q config as bool: %w", configLocal, err)
+	}
+	var workLoop runner
+	if local {
```

Comment:
nit: the condition can be inverted so that you can unconditionally return and then the `else` branch and `var workLoop` declaration above are not necessary

Reply:
Good point.